nautobot/extras/templates/extras/jobresult.html

```diff
@@ -160,6 +160,30 @@
     {% endif %}
 {% endblock extra_tab_content %}
 
+{% block advanced_content_right_page %}
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <strong>Worker</strong>
+    </div>
+    <table class="table table-hover panel-body attr-table">
+        <tbody>
+            <tr>
+                <td>Worker Hostname</td>
+                <td>{{ result.worker }}</td>
+            </tr>
+            <tr>
+                <td>Task Name</td>
+                <td>{{ result.task_name }}</td>
+            </tr>
+            <tr>
+                <td>Meta</td>
+                <td>{% include 'extras/inc/json_data.html' with data=result.meta format="json" %}</td>
```

Comment:
Any idea what sorts of information this field may contain? I haven't ever looked at it.

Reply:
I have no idea... Honestly I wasn't really sure what to put here other than just the Hostname, as we really don't get much data with Celery. I can remove Meta as we don't currently use it in Nautobot.